### PR TITLE
Remove containers from production code

### PR DESCRIPTION
**Pull request by kariuwu**

### CODE REVIEW

File: infinispan/README.adoc [hunk 1]
```diff
@@ -5,19 +5,16 @@
...
-
-You can build this example using:
-
-    $ mvn package
 
 === Run
 
 You can run this example using:
 
-    $ mvn spring-boot:run
```
Comment (oscerd) [inline infinispan/README.adoc:18]: Without this the app won't run, you'll just spin a container.

**Review by oscerd — CHANGES_REQUESTED** [review]
I think this is incomplete.

**Review by oscerd — CHANGES_REQUESTED** [review]
This is not production code, just an example.

**Comment by davsclaus** [comment]
closing old and out-dated PRs

**Pull request closed without merging**